core/src/test/java/org/apache/calcite/test/SqlFunctionsTest.java

```diff
@@ -79,6 +80,12 @@
     assertEquals("nullb", concat(null, "b"));
   }
 
+  @Test public void testStrcmp() {
+    assert -1 == strcmp("text", "text2");
+    assert 1 == strcmp("text2", "text");
+    assert 0 == strcmp("text", "text");
```

Comment:
Can we also add some negative tests ? The invalid arguments and `binary/varbinary` type arguments.

Reply:
the strcmp funciton has two arguments with string type.